addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -86,11 +86,42 @@ $o-wslides-fs-side-width: 300px;
     .o_wslides_js_lesson_quiz {
         i.o_wslides_js_quiz_icon {
             cursor: pointer;
+            @media (max-width: 600px){
+                margin-right: inherit;
+            }
         }
 
         i.o_wslides_js_quiz_icon:hover {
             color: black !important;
         }
+
+        @media (max-width: 600px){
+            .o_wslides_js_quiz_add_answer {
+                margin-left: 6px;
+            }
+            .o_wslides_js_quiz_remove_answer {
+                margin-left: 6px;
+            }
```

Comment:
We try to avoid specifying pixels for sizes in favor of "rem".

Here, `margin-left: .5rem;` should do the trick.

You can also group the classes.

```
.o_wslides_js_quiz_add_answer, .o_wslides_js_quiz_remove_answer {
    margin-left: .5rem;
}
```